core/gui/src/app/workspace/service/workflow-result-export/workflow-result-export.service.spec.ts

```diff
@@ -193,7 +193,8 @@ describe("WorkflowResultExportService", () => {
 
     downloadServiceSpy.downloadOperatorsResult.and.returnValue(of(new Blob()));
```

Comment:
The name of the service is called "DownloadService" and we have to change it to "ExportService". Let me work on it later.